projects/kalidea/kaligraphi/src/lib/atoms/kal-checkbox/kal-checkbox.component.spec.ts

```diff
@@ -1,25 +1,92 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ElementRef, NO_ERRORS_SCHEMA } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
 
 import { KalCheckboxComponent } from './kal-checkbox.component';
+import { FormElementComponent } from '../../utils';
 
-describe('KalCheckboxComponent', () => {
+fdescribe('KalCheckboxComponent', () => {
   let component: KalCheckboxComponent;
   let fixture: ComponentFixture<KalCheckboxComponent>;
+  let elementRef: ElementRef;
+  let checkbox;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ KalCheckboxComponent ]
+      imports: [ReactiveFormsModule],
+      declarations: [
+        KalCheckboxComponent,
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
     })
-    .compileComponents();
+      .createComponent(KalCheckboxComponent);
```

Comment:
duplication of `createComponent(KalCheckboxComponent)`: in `beforeEach(async` and `beforeEach(`.
Replace by original `compileComponents()`